fecgov/openFEC
# Added test_case for 5 endpoints in audit.

1)Refactor models name: AuditPrimaryCategory and AuditCategory
make model name distinctive from other endpoints

2)add 5 test cases in test_audit.py

How to test the changes locally?
a)setup openFEC locally.
b) test 5 endpoints in audit work well.
c)run: `py.test tests/test_audit.py`
and `py.test`
make sure all tests pass.
